data/Person.csv

```diff
@@ -331,12 +331,12 @@ P0182,列寧,弗拉基米尔,zh,male,NULL,NULL,NULL,NULL,1870,1924,弗拉基米
 P0182,Ленин,Владимир,ru,male,NULL,NULL,NULL,NULL,1870,1924,Влади́мир Ильи́ч Улья́нов,ru,PL0094,NULL,NULL,https://en.wikipedia.org/wiki/Vladimir_Lenin,,,,,,2017-10-24,MG,2017-12-23,MG
 P0183,Li,Ruzhen,en,male,NULL,NULL,NULL,NULL,1763,1830,Songsh Daoren,en,PL0002,NULL,NULL,https://en.wikipedia.org/wiki/Li_Ruzhen,,,,,,2017-10-25,MG,2017-12-23,MG
 P0183,李,汝珍,zh,male,NULL,NULL,NULL,NULL,1763,1830,松石道人,zh,PL0002,NULL,NULL,https://en.wikipedia.org/wiki/Li_Ruzhen,,,,,,2017-10-25,MG,2017-12-23,MG
-P0184,Fan,Zhen,en,male,NULL,NULL,NULL,NULL,450,515,NULL,NULL,PL0097,NULL,NULL,https://en.wikipedia.org/wiki/Fan_Zhen,,,,,,2017-10-30,MG,2017-12-23,MG
-P0184,范,缜,zh,male,NULL,NULL,NULL,NULL,450,515,NULL,NULL,PL0097,NULL,NULL,https://en.wikipedia.org/wiki/Fan_Zhen,,,,,,2017-10-30,MG,2017-12-23,MG
-P0185,Yi,Jia,zh,male,NULL,NULL,NULL,NULL,200,169,Yi Sheng,en,PL0077,NULL,NULL,https://en.wikipedia.org/wiki/Jia_Yi,,,,,,2017-10-30,MG,2017-12-23,MG
-P0185,贾,谊,en,male,NULL,NULL,NULL,NULL,200,169,贾生,zh,PL0077,NULL,NULL,https://en.wikipedia.org/wiki/Jia_Yi,,,,,,2017-10-30,MG,2017-12-23,MG
-P0186,Kongzi,,en,male,NULL,NULL,NULL,NULL,551,479,Kong Fuzi,en,PL0098,NULL,NULL,https://de.wikipedia.org/wiki/Konfuzius,,,,,,2017-10-30,MG,2017-12-23,MG
-P0186,孔子,,zh,male,NULL,NULL,NULL,NULL,551,479,孔夫子,zh,PL0098,NULL,NULL,https://de.wikipedia.org/wiki/Konfuzius,,,,,,2017-10-30,MG,2017-12-23,MG
+P0184,Fan,Zhen,en,male,NULL,NULL,NULL,NULL,0450,0515,NULL,NULL,PL0097,NULL,NULL,https://en.wikipedia.org/wiki/Fan_Zhen,,,,,,2017-10-30,MG,2017-12-23,MG
+P0184,范,缜,zh,male,NULL,NULL,NULL,NULL,0450,0515,NULL,NULL,PL0097,NULL,NULL,https://en.wikipedia.org/wiki/Fan_Zhen,,,,,,2017-10-30,MG,2017-12-23,MG
+P0185,Yi,Jia,zh,male,NULL,NULL,NULL,NULL,0200,0169,Yi Sheng,en,PL0077,NULL,NULL,https://en.wikipedia.org/wiki/Jia_Yi,,,,,,2017-10-30,MG,2017-12-23,MG
```

Comment:
Unless Yi pulled a Benjamin Button these are BCE dates so they need the leading `-`